### PR TITLE
Added modifyActiveCountOnInvalidatedSession property to metatype.xml

### PR DESCRIPTION
**Pull request by randyclee**
Fixes: #12158

#build

### CODE REVIEW

**Comment by randyclee** [comment]
#build

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by ellen-lau** [comment]
#run-libby-bot

**Comment by ellen-lau** [comment]
**DO NOT MERGE** until after 20006 GM release date May-27, this is going out in 20007

**Comment by zhang-ca** [comment]
Looks good. Thanks.

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -133,4 +133,7 @@ advanced.name=Advanced
 advanced.description=The default values of these options work for most environments.
 
 extraProperties.name=Additional properties
-extraProperties.description=Additional properties related to session management.
\ No newline at end of file
+extraProperties.description=Additional properties related to session management.
+
+modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric may become inaccurate when the session is accessed by multiple applications, set the property to "false" to resolve the issue.
```
Comment (helyarp) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties:139]: Suggested changes:
- Change "may" to "might"
- Break the text into 2 sentences. Thus, change  "applications, set" to "applications. Set"

**Comment by ellen-lau** [comment]
#run-libby-bot

**Pull request merged**